crates/solvers/src/domain/eth/mod.rs

```diff
@@ -45,6 +45,16 @@ impl From<U256> for Ether {
     }
 }
 
+/// A token amount in wei, always representing the sell token of an order.
+#[derive(Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd)]
+pub struct SellTokenAmount(pub U256);
+
+impl From<U256> for SellTokenAmount {
```

Comment:
This `From` implementation can be avoided by using the crate `derive_more`, I added it to the crates workspace, so it shouldn't be any size overhead to add it to this create too. Then we can just import:
```rust
use derive_more::{Into, From};

#[derive(Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd, From, Into)]
pub struct SellTokenAmount(pub U256);
```

Reply:
Used, thanks